packages/ra-ui-materialui/src/Labeled.tsx

```diff
@@ -47,7 +48,13 @@ export const Labeled = ({
         children.type?.displayName !== 'Labeled' &&
         // @ts-ignore
         children.type?.displayName !== 'Labeled' ? (
-            <Typography color={color} className={LabeledClasses.label}>
+            <Typography
+                sx={{
+                    color: theme =>
+                        get(theme.palette, color.toString(), color).toString(),
+                }}
```

Comment:
This ensure we keep backward compatibility with v5. Indeed, `Typography` does not support the `color` prop anymore (https://mui.com/material-ui/migration/upgrade-to-v6/#typography)

Reply:
Actually no, you had to pass `color="primary.main"` already before this change:

Reply:
I just tested it and, no, it used to work with both. So your implementation is a regression, sorry :slightly_frowning_face: